configure

```diff
@@ -77,12 +80,24 @@ RCOBJS=
 STRIP=
 ARCHS=
 PC_CFLAGS=
-prefix=${prefix-/usr/local}
+if test "${uname}" = "Haiku" || test "${uname}" = "haiku"; then
+  prefix=${prefix-/boot/system}
```

Comment:
The Haiku equivalent of `/usr/local` is `/boot/system/non-packaged`. `/boot/system` only works for packages, because it is read-only.

Reply:
This is just an early patch, to get configure and cmake make identical binaries, after that I start working on package recipe. Supporting both non-packaged and packaged build makes things too complicated and suits better for downstream patch if needed.